src/sgame/sg_main.cpp

```diff
@@ -2273,6 +2276,10 @@ can see the last frag.
 */
 void CheckExitRules()
 {
```

Comment:
Just a side note, is it possible to comment under the line instead of above it? That's what GitHub expects and it always display previous line and drop following lines in comment thread so the related code is not displayed when a comment is set above the line.